src/main.cpp

```diff
@@ -15,6 +15,7 @@ int main(int argc, char **argv) {
   }
 #endif
 
+  QApplication::setStyle(QStyleFactory::create("Fusion"));
```

Comment:
If I were you, I'd surround this with `ifdef Q_OS_MACOS` since that's where this happens